examples/fluids/problems/newtonian.c

```diff
@@ -98,7 +99,7 @@ PetscErrorCode NS_NEWTONIAN_IG(ProblemData *problem, DM dm, void *ctx, SimpleBC
   CeedScalar lambda = -2. / 3.;       // -
   CeedScalar mu     = 1.8e-5;         // Pa s, dynamic viscosity
   CeedScalar k      = 0.02638;        // W/(m K)
-  CeedScalar c_tau  = 0.5;            // -
+  CeedScalar c_tau  = 0.5 / degree;   // -
   CeedScalar Ctau_t = 1.0;            // -
   CeedScalar Ctau_v = 36.0;           // TODO make function of degree
```

Comment:
This is what we call `Ctau_v`. I'd rather have a function that works for arbitrary order, or at least theoretical justification for these values, but just indexing this sequence `[36, 60, 128]` would be pragmatic and it's not clear we'll want to go higher than cubic anyway. https://github.com/PHASTA/phasta/blob/master/phSolver/compressible/e3tau.f#L146-L154

Reply:
Can we automate other `Ctau`s as well or should I drop the TODO comments?

Reply:
Please just set them all like is done in blasius.yaml and the PHASTA link above, for degree 1, 2, 3. Does that make sense?

Reply:
Hmm, looking at blasius.yaml, I would guess that `Ctau_C = Ctau_M = 0.25 / degree` and `Ctau_E = 0.125`, is that right?